Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Definitions/Result.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.ComponentModel;
+
+namespace Frends.Excel.ConvertToJSON.Definitions;
```

Comment:
Same here.

Reply:
In C# best practices namespaces go according to root namespace + folder structure.